book/src/external-attacher.md

```diff
@@ -10,16 +10,21 @@
+[external-attacher v4.2.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v4.2.0) | [release-4.2](https://github.com/kubernetes-csi/external-attacher/tree/release-4.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-attacher:v4.2.0 | v1.17 | - | v1.22
+[external-attacher v4.1.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v4.1.0) | [release-4.1](https://github.com/kubernetes-csi/external-attacher/tree/release-4.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-attacher:v4.1.0 | v1.17 | - | v1.22
+[external-attacher v4.0.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v4.0.0) | [release-4.0](https://github.com/kubernetes-csi/external-attacher/tree/release-4.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-attacher:v4.0.0 | v1.17 | - | v1.22
+[external-attacher v3.5.1](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.5.1) | [release-3.5](https://github.com/kubernetes-csi/external-attacher/tree/release-3.5) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-attacher:v3.5.1 | v1.17 | - | v1.22
 
 ### Unsupported Versions
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-attacher v3.1.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.1.0) | [release-3.1](https://github.com/kubernetes-csi/external-attacher/tree/release-3.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.1.0 | v1.17 | - | v1.17
-[external-attacher v3.0.2](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.0.2) | [release-3.0](https://github.com/kubernetes-csi/external-attacher/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.0.2 | v1.17 | - | v1.17
+[external-attacher v3.5.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.5.0) | [release-3.5](https://github.com/kubernetes-csi/external-attacher/tree/release-3.5) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-attacher:v3.5.0 | v1.17 | - | v1.22
```

Comment:
I think we generally only list the latest patch version in a minor branch, so this can be removed. Ditto elsewhere.

Reply:
Removed.